server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSLexer.g4

```diff
@@ -851,6 +857,8 @@ TOCHANNEL : T O C H A N N E L;
 TOCONTAINER : T O C O N T A I N E R;
 TOFLENGTH : T O F L E N G T H;
 TOKEN : T O K E N;
+TOKENLEN : T O K E N L E N;
+TOKENTYPE : T O K E N T Y P E;
```

Comment:
Verify if the new tokens should be included in the `cicsLexerDefinedVariableUsageTokens` rule.